src/content/1.7/development/components/form/types-reference/integer-min-max-filter.md

```diff
@@ -0,0 +1,29 @@
+---
+title: IntegerMinMaxFilterType
+menuTitle: IntegerMinMaxFilterType
+weight: 2
```

Comment:
Weight is used for ordering. If all elements have the same weight then it's useless :)

Reply:
Thank you - had to do it for all form type-reference level .md files to keep them all aligned alphabetically . https://github.com/PrestaShop/docs/pull/305/commits/8d18a6047d4e7fe995a4ece4835e42ebfb35d423